src/main/java/com/example/artvswar/config/AppConfig.java

```diff
@@ -38,4 +39,9 @@ public void addCorsMappings(CorsRegistry registry) {
     public WebClient buildWebClient() {
         return WebClient.builder().build();
     }
+
+    @Bean
```

Comment:
As far as I know you don't need to create bean of RestClient, it is provided by default by Spring